pkg/gadgets/socket-collector/tracer/bpf/socket-common.h

```diff
@@ -38,7 +38,7 @@
  * Returns:
  * * The inode identifier associated to the socket.
  */
-static long sock_i_ino(const struct sock *sk)
+static unsigned long sock_i_ino(const struct sock *sk)
```

Comment:
Could you please squash this modification and the fixup commit?

Reply:
Oops! Good catch, it should be fixed now!